src/components/RoutePath.tsx

```diff
@@ -0,0 +1,24 @@
+import { Route, Routes } from 'react-router-dom';
+import Home from './Home';
+import { RoutePathProps } from '../types/component-types';
+
+const routes = [
+  {
+    path: '/',
+    element: <Home />,
+  },
+];
+
+const RoutePath: React.FC<RoutePathProps> = () => (
+  <Routes>
+    {routes.map((route, index) => (
```

Comment:
while this is ok, let's not map and actually write it out so we can simply do the nested routes. We can always refactor to a map later. You can remove the routes object and just type out every route.